apps/condo/domains/common/utils/codegeneration/generate.hooks.ts

```diff
@@ -0,0 +1,220 @@
+import { useCallback } from 'react'
```

Comment:
We have a module `'@condo/codegen/generate.hooks'` for that.
Why this code is here?